### ISSUE TITLE
Link users to sent folder in confirmation alert

### ISSUE THREAD

**Issue by beckyphung** [department-of-veterans-affairs/va.gov-team#124122]
## Overview
At the 11/3 EHRM coordinators call, Sylvia suggested linking back to the sent folder in the confirmation alert. The only relevant Jira ticket I could find is this empty one: [Accessibility - sent messages easier to find](https://jira.devops.va.gov/browse/MHV-57571?jql=project%20%3D%20MHV%20AND%20resolution%20%3D%20Unresolved%20AND%20text%20~%20%22Accessibility%20-%20sent%20messages%20easier%20to%20find%22%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC)

<img width="978" height="295" alt="Image" src="https://github.com/user-attachments/assets/7efbcfa8-9e67-4dfb-af0e-611a2e8a7079" />

## Acceptance criteria 
- [ ] we understand the extent to this is a challenge for users
   - need more info on the feedback sylvia's heard

## Tasks
- [x] ask Sylvia for more feedback 
- [x] UX to confirm content
- [ ] engineering implementation
- [ ] document this feedback

**Comment by thomsonespy** [comment]
<img width="635" height="122" alt="Image" src="https://github.com/user-attachments/assets/98c679ae-d456-46a4-aca2-5bcc34570177" />

@6StringCode I confirmed with content and here is the updated content. The type of link seems conflicting based on VADS standards. It seems like most links in alerts should be action or active links, but because there is not other text I am making this a standard link.

**Comment by thomsonespy** [comment]
Historical context captured for this ticket:

"it sounds like at least part of the decision to place the alert on top was in order to maintain voice over hierarchy and ensure that after calling out the alert, the user would still be able to tab forward to the h1 and understand where they were (what page). I sounds like in some past iterations, there were complications with having the alert appropriately announced and then still being able to catch the h1 if the alert is below it – the VO would just skip over the h1.  I thought maybe using the assertive prop for the aria would fix this, but it sounds it isn't the correct fix and can open up other cans of worms.
TBH, the least amount of engineering work is probably to leave the positioning as is, the text will just be a semantically agnostic <p> tag, and then we style it however we want it styled. We can style it like an <h2> if we want."

**Comment by 6StringCode** [comment]
> Can the text in the message sent from sent file be a slim alert > meaning that the text is not a H2 style?

Yep! Sorry I missed that. I also caught and fixed another edge case in which a user _replying_ from a sent message will also NOT see the link since this requires different logic coming from a thread than from the compose flow.

<img width="946" height="596" alt="Image" src="https://github.com/user-attachments/assets/f84cd76b-c1ac-46b8-8781-f7d79af45800" />

**Comment by sarahhortonadhoc** [comment]
Chiming in here, I agree with [@thomsonespy's comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/124122#issuecomment-3699386233) about positioning alerts after the level 1 heading.
